helmfile.d/snippets/rawValues.gotmpl

```diff
@@ -0,0 +1 @@
+{{- .Environment.Values | get (printf "charts.%s._rawValues" .Release.Name) dict | toYaml }}
```

Comment:
Why provide a dict here? Would nil not be sufficient?

Reply:
No, because toYaml will fail.
> toYaml marshals a map into a string